Dockerfile

```diff
@@ -55,12 +55,13 @@ COPY evm_loader/test_requirements.txt solana-py.patch /tmp/
 RUN pip3 install -r /tmp/test_requirements.txt
 RUN cd /usr/local/lib/python3.8/dist-packages/ && patch -p0 </tmp/solana-py.patch
 
-COPY --from=solana /opt/solana/bin/solana /opt/solana/bin/solana-keygen /opt/solana/bin/solana-faucet /opt/solana/bin/
-COPY --from=solana-deploy /opt/solana/bin/solana /opt/solana/bin/solana-deploy
+# COPY --from=solana /opt/solana/bin/solana /opt/solana/bin/solana-keygen /opt/solana/bin/solana-faucet /opt/solana/bin/
+COPY --from=solana  /opt/solana/bin/solana-keygen /opt/solana/bin/solana-faucet /opt/solana/bin/
```

Comment:
Since we no longer need a custom build of the Solana CLI, we can use it from an image with a validator node. Remove `cybercoredev/solana:latest` image and use `solana` from `cybercoredev/solana:v1.6.7-resources`.

Reply:
fixed